### DISCUSSION TITLE
nama repo nya kenapa nggak cuyuniverse juga?

### DISCUSSION THREAD

**Discussion by lamualfa** [deaafrizal/cuyuniverse-forum#77] [Q&A] [answered]
nama repo nya kenapa nggak cuyuniverse juga?

**Comment by deaafrizal** [comment, marked as answer]
ketinggalan bre, ini awalnya kan projek yang buat konten mencoba laravel react. eh keterusan

**Reply by lamualfa** [reply]
kenapa nggak direname aja bang? atau memang sengaja.

**Reply by deaafrizal** [reply]
udah gue tempel ke server, kalau ubah nama repo mesti setup beberapa hal lagi di server
